#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_gist_man_typo.dpatch by Thibaut Paumard <paumard@users.sourceforge.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix a typo ('supress') in gist manpage. Submitted to upstream.

@DPATCH@

diff -urN orig/doc/gist.1 tp/doc/gist.1
--- orig/doc/gist.1	2006-05-19 09:47:11.968070680 +0200
+++ tp/doc/gist.1	2006-05-19 09:47:59.279878192 +0200
@@ -67,7 +67,7 @@
 gist window using your window manager or -geometry.
 .TP
 -nd
-to supress the automatic display of the first
+to suppress the automatic display of the first
 picture when gist starts in interactive mode
 .TP
 -b
